app/views/groups/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="group-page">
+
+  <div class="group-items-container">
+    <% @groups.each do |group| %>
+
+      <%= render group %>
+
+    <% end %>
+  </div>
+
+  <div class='new-group-link' >
+  <%= link_to 'NEW GROUP', new_user_group_path(user_id: current_user.id), class: 'new-group-link-id' %>
+  </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you if you change this button name to something like `New Category` or `Add Category` to be meaningful according to page name.